src/screens/Main.tsx

```diff
@@ -53,12 +54,6 @@ const MenuContents = [
     title: 'Custom Call',
     isEnabled: true,
     component: <CustomCall />
-  },
-  {
```

Comment:
Oh no! why would you do that? This is used in a different PR and should be part of the dropdown menu

Reply:
Which PR? Do we really need it? Doesn't seem to be the case for me. Should I revert changes?

Reply:
> Which PR? Do we really need it? Doesn't seem to be the case for me. Should I revert changes?

Hm... Nah - Just leave it and we'll fix it after merge